### ISSUE TITLE
Any one test the accuracy on VOC dataset?

### ISSUE THREAD

**Issue by blueardour** [ShuangXieIrene/ssds.pytorch#41]

**Comment by Brozhao** [comment]
Did you test the mAP of SSD+VGG+VOC ? Mine is just 0.47

**Comment by Brozhao** [comment]
also my ssd_lite_mobilenetv2_voc is just 0.66

**Comment by blueardour** [comment]
I only focus on mobilenetv2. Which pytorch version and python version do you employ?

**Comment by blueardour** [comment]
Yes, pre-trained model was used. My environment is python2.7/3.6 and pytorch 0.4.1

**Comment by Brozhao** [comment]
Could you tell me the learning rate and max epoch of yours ?   Thanks a lot!

**Comment by rw1995** [comment]
> Could you tell me the learning rate and max epoch of yours ? Thanks a lot!

do you slove?
